src/EA.Weee.RequestHandlers/Admin/AatfReports/GetAatfAeReturnDataCsvHandler.cs

```diff
@@ -56,11 +56,9 @@ public async Task<CSVFileData> HandleAsync(GetAatfAeReturnDataCsv request)
 
             //Trim the space before equals in  =Hyperlink
             fileContent = fileContent.Replace(" =HYPERLINK", "=HYPERLINK");
-
-            var fileName = string.Format("{0}_{2}_RETURN_{1:ddMMyyyy_HHmm}_Q{3}.csv",
+              var fileName = string.Format("{0}_Q{2}_Summary_of_AATF-AE returns to date_{1:ddMMyyyy}_{1:HHmm}.csv",
```

Comment:
Not related this change, but can we change this to use SystemTime.UtcNow and then update the unit test to ensure that the date time part is correct...